allenact/algorithms/onpolicy_sync/vector_sampled_tasks.py

```diff
@@ -13,27 +13,26 @@
 from typing import (
     Any,
     Callable,
+    Dict,
+    Generator,
+    Iterator,
     List,
     Optional,
     Sequence,
     Set,
     Tuple,
     Union,
-    Dict,
-    Generator,
-    Iterator,
     cast,
 )
 
 import numpy as np
-from gym.spaces.dict import Dict as SpaceDict
-from setproctitle import setproctitle as ptitle
-
 from allenact.base_abstractions.misc import RLStepResult
 from allenact.base_abstractions.task import TaskSampler
 from allenact.utils.misc_utils import partition_sequence
 from allenact.utils.system import get_logger
 from allenact.utils.tensor_utils import tile_images
+from gym.spaces.dict import Dict as SpaceDict
+from setproctitle import setproctitle as ptitle
```

Comment:
Is this actually used in this file?

Reply:
It is, [see line 300](https://github.com/allenai/allenact/pull/339/files#diff-8956c84e3e4a34b77af776d8616dabc3fda44b886cd8062b2b2ffca14f0fe73eR300) (hidden by default by GitHub).